bundle.Dockerfile

```diff
@@ -1,7 +1,5 @@
 FROM scratch
 
-ARG USER_ID=1001
```

Comment:
Are these intended changes? I think this file is re-generated automatically during `make`

Reply:
Yes, it's autogenerated by the `make bundle` command. We added the user manually, but since the bundle image is just an index image, it's probably not needed. Checked a few other operators and they don't add the user either.